tests/opts.rs

```diff
@@ -46,25 +52,746 @@ fn opts_using_long_equals() {
 
 #[test]
 fn opts_using_mixed() {
-    let m = App::new("opts")
-        .args(vec![
+    let r = App::new("opts")
+        .args(&[
             Arg::from_usage("-f, --flag [flag] 'some flag'"),
             Arg::from_usage("-c, --color [color] 'some other flag'")
             ])
-        .get_matches_from(vec!["", "-f", "some", "--color", "other"]);
+        .get_matches_from_safe(vec!["", "-f", "some", "--color", "other"]);
+    assert!(r.is_ok());
+    let m = r.unwrap();
     assert!(m.is_present("flag"));
     assert_eq!(m.value_of("flag").unwrap(), "some");
     assert!(m.is_present("color"));
     assert_eq!(m.value_of("color").unwrap(), "other");
+}
 
-    let m = App::new("opts")
-        .args(vec![
+#[test]
+fn opts_using_mixed2() {
+    let r = App::new("opts")
+        .args(&[
             Arg::from_usage("-f, --flag [flag] 'some flag'"),
             Arg::from_usage("-c, --color [color] 'some other flag'")
             ])
-        .get_matches_from(vec!["", "--flag=some", "-c", "other"]);
+        .get_matches_from_safe(vec!["", "--flag=some", "-c", "other"]);
+    assert!(r.is_ok());
+    let m = r.unwrap();
     assert!(m.is_present("flag"));
     assert_eq!(m.value_of("flag").unwrap(), "some");
     assert!(m.is_present("color"));
     assert_eq!(m.value_of("color").unwrap(), "other");
-}
\ No newline at end of file
+}
+
+#[test]
```

Comment:
Perhaps, but we can shuffle around where the test lies at a later date when we re-organize some things. :stuck_out_tongue_winking_eye: